pes-sa-remove/pes-sa-remove.php

```diff
@@ -0,0 +1,141 @@
+<?php
+### Copyright 1999-2024. WebPros International GmbH.
+
+###############################################################################
+# This script shows addresses in spamfilter and spamfilter_preference that are not present in Plesk.
+# Requirements : >php7.4, GNU coreutils
+# Version      : 1.1
+#########
+$short_options = "fh";
```

Comment:
There are a lot of code-style violations. Please follow: https://www.php-fig.org/psr/psr-1/ and https://www.php-fig.org/psr/psr-12/